### DISCUSSION TITLE
Anybody managed to run this on free T4 GPU google colab?

### DISCUSSION THREAD

**Discussion by tralala87** [vladmandic/sdnext#1735] [Q&A]
Anybody managed to run this on free T4 GPU google colab?

**Comment by TeutonJon78** [comment]
Google starting blocking stable diffusion on the free colab's a few weeks ago. You have to use a paid tier now.

**Reply by tralala87** [reply]
but isn't the webui from vladmandic able to avoid this? I thought this blockage is only for automatic111 webui

**Comment by ash368** [comment]
> but isn't the webui from vladmandic able to avoid this? I thought this blockage is only for automatic111 webui

i think colab blocking cell containing words "SD", "stable diffusion"
